Backpack/src/main/res/values/attrs.xml

```diff
@@ -110,6 +110,11 @@
       <enum name="small" value="0" />
       <enum name="large" value="1" />
     </attr>
+    <attr name="elevationLevel" format="enum">
```

Comment:
can't use elevation or cardElevation as a name as that already exists in material